website/content/en/docs/getting-started/_index.md

```diff
@@ -1,21 +1,82 @@
 ---
 title: Getting Started
-description: Trying ClusterLink for yourself
 weight: 20
 ---
 
+This guide will give you a quick start on installing and setting up the ClusterLink project on a Kubernetes cluster.
+
 ## Prerequisites
 
-<!-- Are there any system requirements for using your project? What languages are supported (if any)? Do users need to already have any software or tools installed? -->
+Before you start, you must have access to a Kubernetes cluster. For example, you can set up a local environment using the [Kind](https://kind.sigs.k8s.io/) project.
 
 ## Installation
 
-<!-- Where can your user find your project code? How can they install it (binaries, installable package, build from source)? Are there multiple options/versions they can install and how should they choose the right one for them? -->
+1. To install ClusterLink on Linux or Mac, use the installation script:
+
+        curl -L <https://Clusterlink.net/install> | sh -
```

Comment:
surround shell command with backticks and `sh` indication?
`curl` is a prerequisite as is `kubectl`?

Reply:
Yes, but Curl is basic, I didn't see any other instructions that guide you to install Curl.